### PR TITLE
created home page

### PR DESCRIPTION
**Pull request by finlaysaunders**

### CODE REVIEW

File: pages/home.html [hunk 1]
```diff
@@ -2,9 +2,31 @@
 <html lang="en" dir="ltr">
   <head>
     <meta charset="utf-8">
+    <link rel="stylesheet" href="../css/bootstrap.css">
+    <link rel="stylesheet" href="../css/master.css">
     <title>Home Page</title>
   </head>
-  <body>
+  <body id=homeBackground>
```
Comment (ethanedwards115) [inline pages/home.html]: Instead of putting an id on the body element, try using a `<main>` tag with the same id.

Comment (finlaysaunders) [reply pages/home.html]: will do

File: pages/home.html [hunk 2]
```diff
@@ -2,9 +2,31 @@
...
+
+    <h1 id=homeHeading>Live well <br> in Dundee</h1>
```
Comment (ethanedwards115) [inline pages/home.html]: try not to use `<br>` in your html code. Just use the id to make the container smaller, or use Bootstrap's `.col-3` or a smaller column if necessary

**Review by ethanedwards115 — CHANGES_REQUESTED** [review]
I've left comments on the necessary lines of code which need to be changed before I can approve a merge

File: pages/home.html (hunk 1 above)
Comment (finlaysaunders) [reply pages/home.html]: done

File: pages/home.html (hunk 2 above)
Comment (ethanedwards115) [reply pages/home.html]: `.col-3` is a class so try not to put it in the id section
try this: `<h1 class="col col-3" id="homeHeading">Living well in Dundee<h1>`
feel free to change the column size to fit the proportions of the text

Comment (finlaysaunders) [reply pages/home.html]: used container class

File: pages/home.html [hunk 3]
```diff
@@ -2,9 +2,47 @@
...
     <meta charset="utf-8">
+    <link rel="stylesheet" href="../css/bootstrap.css">
+    <link rel="stylesheet" href="../css/master.css">
+    <script src="https://kit.fontawesome.com/6854d7f83d.js" crossorigin="anonymous"></script>
     <title>Home Page</title>
   </head>
-  <body>
+
+  <main id=homeBackground>
+    <body >
```
Comment (ethanedwards115) [inline pages/home.html]: Swap these tags around, please.

Comment (finlaysaunders) [reply pages/home.html]: main inside body?

File: pages/home.html [hunk 4]
```diff
@@ -2,9 +2,47 @@
...
+  <footer>
+    <div class="footer">
+      <div class="icons">
+        <a href="https://www.facebook.com/thelittlegreenlarder/" target="_blank"><em class="fab fa-facebook-square fa-2x"></em></a>
+        <a href="https://www.instagram.com/thelittlegreenlarder/" target="_blank"><em class="fab fa-instagram fa-2x"></em></a>
+     </div>
+    </div>
+  </footer>
+
+</main>
```
Comment (ethanedwards115) [inline pages/home.html]: You should end the `<main>` tag before you end the `<body>` tag.
Keep the footer within the `<body>` tag as well

Comment (finlaysaunders) [reply pages/home.html]: ok, done

File: pages/home.html (hunk 3 above)
Comment (ethanedwards115) [reply pages/home.html]: Yeah, like this:
```
<body>
  <main>
    <p>hello world!</p>
  </main>
</body>
```

Comment (finlaysaunders) [reply pages/home.html]: are the cards ok inside of <main> as well?

Comment (finlaysaunders) [reply pages/home.html]: main

**Review by ethanedwards115 — APPROVED** [review]
This looks good now, thanks for making those changes!

**Comment by finlaysaunders** [comment]
thanks for helping, always learning!

**Pull request merged**